arroyo-api/src/jobs.rs

```diff
@@ -269,11 +270,17 @@ pub async fn get_job_errors(
 
 impl Into<JobLogMessage> for DbLogMessage {
     fn into(self) -> JobLogMessage {
+        let level: JobLogLevel = match self.log_level {
+            LogLevel::info => JobLogLevel::Info,
+            LogLevel::warn => JobLogLevel::Warn,
+            LogLevel::error => JobLogLevel::Error,
+        };
```

Comment:
I had to do this casting inline since it's a database type, which arroyo-rpc doesn't have access to. Same with `StopMode` below.